packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/implicit/use-flow.md

```diff
@@ -2,8 +2,7 @@ Without using existing libraries, you can make a direct request to Okta's [OIDC
 
 ### Request for tokens
 
-Using this flow is very similar to the <GuideLink link="../../authcode/main/">authorization code flow</GuideLink>
-except that the `response_type` is `token` and/or `id_token` instead of `code`.
+Using this flow is very similar to the [Authorization Code flow](/docs/guides/implement-grant-type/authcode/main/#authorization-code-flow) except that the `response_type` is `token` and/or `id_token` instead of `code`.
```

Comment:
```suggestion
This flow is very similar to the [Authorization Code flow](/docs/guides/implement-grant-type/authcode/main/#authorization-code-flow), except that the `response_type` is `token` and/or `id_token` instead of `code`.
```